### ISSUE TITLE
Split application components into individual containers

### ISSUE THREAD

**Issue by pangelo** [altLab/liquid-feedback-docker#9]
This will make it easier to debug and upgrade each component individually, and eventually allow to run these images in production settings. Initial plan is to split into backend and frontend, and to base of available official images (e.g. postgresq).

**Comment by debuglevel** [comment]
I suggest split up into 3 images: frontend, backend and postgres

**Comment by pangelo** [comment]
I agree, but I'm leaving this till after LF v4 is done, because there might be some changes to the build. Currently, there are some build steps that need to access the database, and these will have to be deferred to the container runtime and will require additional scripting of the startup process.

If anyone wants to take a stab at this before v4, I'll gladly accept patches :)
